http-server-netty/src/main/java/io/micronaut/http/server/netty/NettyHttpServer.java

```diff
@@ -687,6 +697,18 @@ private static String displayAddress(NettyHttpServerConfiguration.NettyListenerC
         };
     }
 
+    public static <T> CompletionStage<T> toCompletionStage(Future<T> future) {
```

Comment:
does this need to be public? Should it be moved to utility class? Seems odd it is here